### DISCUSSION TITLE
Icon paths

### DISCUSSION THREAD

**Discussion by grahamford** [vmware/clarity#6719] [Q&A] [answered]
The documentation has this example to use an icon in an Angular app

```
import '@cds/core/icon/register.js';
import { ClarityIcons, userIcon } from '@cds/core/icon';

ClarityIcons.addIcons(userIcon);
```

Not all icons are exported from `@cds/core/icon` however, eg `timelineIcon`. For this I have the import statement

```
import { timelineIcon } from '@cds/core/icon/shapes/timeline';
```

Is this the way newer icons should be imported?

**Comment by bbogdanov** [comment, marked as answer]
Hi @grahamford,

Yea not all icons are been exported as single ones. Some of them are available through collections. We did a PR to export some of the `technology` icons that were only available when you import the collection but if there are others, let us know.
